pkgs/games/pioneer/default.nix

```diff
@@ -1,29 +1,30 @@
-{ fetchFromGitHub, stdenv, autoconf, automake, pkgconfig
-, curl, libsigcxx, SDL2, SDL2_image, freetype, libvorbis, libpng, assimp, libGLU_combined
+{ fetchFromGitHub, stdenv, cmake, pkgconfig
+, curl, libsigcxx, SDL2, SDL2_image, freetype, libvorbis, libpng, assimp, 
+libGLU_combined, glew
```

Comment:
nitpick: please be consistent with the code you're adding to. In this case please move the comma to the next line:
```
, curl, libsigcxx, SDL2, SDL2_image, freetype, libvorbis, libpng, assimp
, libGLU_combined, glew
```